### ISSUE TITLE
Beginner - need help connecting to microsoft sql with already created tables

### ISSUE THREAD

**Issue by bpolasek519** [tiangolo/fastapi#5877]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
None to show yet
```

### Description

Hi, I don't have much code yet as I'm just getting started and am new to coding in general. I have created a dashboard using Dash. I need to take information (which I have already put into JSON format) and put it into tables I have already made using Microsoft SQL Server Management. My boss suggested I look into FastAPI. I have gone through the tutorial but am still just struggling to translate to my needs. Does anyone have any help they can offer or code they can share so I can see an example? Thanks so much!

### Operating System

Windows

### FastAPI Version

0.89.1

### Python Version

3.10

**Comment by odiseo0** [comment]
Unrelated to FastAPI but you can check [SQLAlchemy](https://docs.sqlalchemy.org/en/14/dialects/mssql.html) and by the way, do you need to create an API? Because that is what FastAPI does. If you only need to put some data in a SQL database just create simple scripts with SQLAlchemy

**Comment by bpolasek519** [comment]
> Unrelated to FastAPI but you can check [SQLAlchemy](https://docs.sqlalchemy.org/en/14/dialects/mssql.html) and by the way, do you need to create an API? Because that is what FastAPI does. If you only need to put some data in a SQL database just create simple scripts with SQLAlchemy

Yeah so that's what I started doing. My boss thought that because I was running a web app I wouldn't be able to do a direct script in my app to the database but I have been able to.
